generators.py

```diff
@@ -0,0 +1,25 @@
+# fill in this function
+import types
+from typing import Generator
+
+
+def fib() -> Generator[int]:
```

Comment:
Good that you added type hints for this. But for Generators, [we need to define the types of the items the generator expects to receive as well as the types of the values it returns](https://docs.python.org/3/library/typing.html#corresponding-to-other-types-in-collections-abc) In this case the correct type hint would be `Generator[int, None, None]`

Reply:
Thanks for the correction. Fixed it.